src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -137,19 +161,204 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_TWO), "SXS", "XBX", "SXS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_THREE), "SXS", "SXS", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_FOUR ), "SSS", "SXS", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_FIVE), "X  ", " X ", " BX", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_SIX), "  X", " X ", "XB ", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_SEVEN), "SXS", "SBS", "SSS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_EIGHT),"SSS", "XXX", "SBS", 'S', WOODEN_STICK, 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
+        GameRegistry.addRecipe(new ShapedOreRecipe(
+                new ItemStack(ModBlocks.timberFrames.get(META_FIVE), FOUR_BLOCKS, META_NINE), "   ", "XXX", " B ", 'X',
+                new ItemStack(Blocks.PLANKS, 1, META_FIVE), 'B', ModItems.buildTool));
 
         //Field
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutField, 1),
                                                     " Y ", "X#X", " X ", 'X', WOODEN_STICK, '#', Items.LEATHER, 'Y', Blocks.HAY_BLOCK));
         //Double Fern
-        GameRegistry.addShapelessRecipe(new ItemStack(Blocks.DOUBLE_PLANT, 1, 3), new ItemStack(Blocks.TALLGRASS, 1, 2), new ItemStack(Blocks.TALLGRASS, 1, 2));
-
-
-        //enableInDevelopmentFeatures(enableInDevelopmentFeatures);
-        addSupplyChestRecipes(supplyChests);
+        GameRegistry.addShapelessRecipe(new ItemStack(Blocks.DOUBLE_PLANT, 1, META_THREE), new ItemStack(Blocks.TALLGRASS, 1, META_TWO), new ItemStack(Blocks.TALLGRASS, 1, META_TWO));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)